mne/time_frequency/tests/test_tfr.py

```diff
@@ -548,7 +549,7 @@ def test_plot_joint():
 
     topomap_args = {'res': 8, 'contours': 0, 'sensors': False}
 
-    for combine in ('mean', 'rms', None):
+    for combine in ('mean', 'rms'):
```

Comment:
combine is None by default, so we should test this value too.

Reply:
The docs say " If None, plot one figure per selected channel." That is the intended behavior, although as you describe, not what the function actually does.

Reply:
Are you sure? This looks like `mean` is the default and further down the docs it lists only `mean` and `rms` as possible values. https://github.com/mne-tools/mne-python/blob/f20c60ec23481e8c9888fb9f7da1544378dd2489/mne/time_frequency/tfr.py#L1314

Reply:
Ah, yeas, you are right, I was looking at `tfr.plot()`, not `plot_joint()`